bigquery/google/cloud/bigquery/external_config.py

```diff
@@ -544,7 +553,7 @@ class ExternalConfig(object):
     """Description of an external data source.
 
     Args:
-        source_format :class:`ExternalSourceFormat`:
+        source_format (:class:`ExternalSourceFormat`):
```

Comment:
Do we actually need the `:class:` part with Napoleon-style? I thought we didn't.

Also, let's do the fully-qualified name.

Reply:
The `:class:` part isn't needed to render the link, but if I switch to the full name I'd still like it to show up as just the class name, and the tilde syntax doesn't work without `:class:`.